package.json

```diff
@@ -44,7 +44,7 @@
     "standard": "^16.0.3"
   },
   "dependencies": {
-    "bfx-api-node-models": "^1.4.1",
+    "bfx-api-node-models": "git+https://github.com/avsek477/bfx-api-node-models.git#add-visible-on-hit-hidden-orders",
```

Comment:
needs to be changed after this [PR](https://github.com/bitfinexcom/bfx-api-node-models/pull/77) is merged.